docs/kss/public/script.js

```diff
@@ -45,7 +45,7 @@ $( document ).ready(function(){
 
     // off canvas menu
     $('[data-toggle="offcanvas"]').click(function () {
-        $('.row-offcanvas').toggleClass('active')
```

Comment:
I would like for us to use the bootstrap toggle opposed to custom implementation here.